components/breadcrumb/src/test.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { mount } from 'enzyme';
+import { createSerializer } from 'jest-emotion';
+import * as emotion from 'emotion';
 
 import Breadcrumb from './';
 
+expect.addSnapshotSerializer(createSerializer(emotion));
```

Comment:
I think this should be in setupTests so it only needs to be called once

Reply:
Yea great shout, I'll get that in there.